runtime/j9vm/jvm.c

```diff
@@ -2670,7 +2670,7 @@ deleteDirsFromLibpath(const char *const libpath, const char *const deleteStart,
 
 	newPath = malloc(preLen + delim + postLen + 1);
 	if (NULL == newPath) {
-		fprintf(stderr, "deleteDirsFromLibpath: malloc(%d) failed, aborting\n", preLen + delim + postLen + 1);
+		fprintf(stderr, "deleteDirsFromLibpath: malloc(%zu) failed, aborting\n", preLen + delim + postLen + 1);
```

Comment:
The `z` modifier is not supported on all platforms (e.g. Windows); perhaps a better fix would be to cast the length to `int` to match `%d`.

Reply:
This code is `#if defined(AIXPPC)`

Reply:
> This code is `#if defined(AIXPPC)`

That wasn't obvious from the web interface. Thanks.